test/generate.test.ts

```diff
@@ -149,6 +150,86 @@ describe('generate', function () {
     });
   });
 
+  describe('scaleType', () => {
+    it('should enumerate correct scaletype for quantitative field', () => {
```

Comment:
pick `scaleType` or `scale type` .. don't use `scaletype`.  Same for the rest of this file.